website/docs/addon_motionbuilder_admin.md

```diff
@@ -10,6 +10,24 @@ import versions from '@site/docs/assets/json/Ayon_addons_version.json'
 {versions.MotionBuilder_Badge}
 </ReactMarkdown>
 
-:::note Work in progress
-This part of documentation is still work in progress.
+## Setting your Motion Builder launching version in Application
+
+Users can set up your own Motion Builder version in Application Addon
+You can click **+** button to add new different version.
+
+![AYON Application Motion Builder](assets/motionbuilder_setting_version_application.png)
+
+:::warning Important
+As the integration developed with Motion Builder 2024/2025, some functions may not be working in the version elder than 2024.
 :::
```

Comment:
```suggestion
:::warning Unknown support for Motion Builder 2023 and below
The AYON Motion Builder integration was initially developed with Motion Builder 2024 and 2025. Hence functionality was not tested in older versions than 2024 and may not work.
:::
```

Reply:
Thank you
https://github.com/ynput/ayon-documentation/pull/221/commits/1945360dfe2770f6a251535d13c63e77de156cc5